README.md

```diff
@@ -1,3 +1,48 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS. It consists of two chains:
+- A deployment chain to provision CloudFoundry on AWS using Concourse, Terraform and BOSH
```

Comment:
I am not sure what it means with "deployment chain" but if it is a pipeline, I would suggest we fit to pipeline. 

Concourse has lots of concepts and adding a new word as "chain" does not simplify it.